### PR TITLE
Initial commit that displays Remember Items

### PR DESCRIPTION
**Pull request by marissa27**
Adds reminder model
Adds route reminder with template reminder
Add a root route
Creates a test that look at displaying reminders

## Purpose

Add functionality to see all reminders

## Approach

Code along with some doc help 

### Learning

Taylor code along and docs 

### Test coverage 

closes issue #1 

### Follow-up tasks
-[Issue #2]

### CODE REVIEW

**Comment by marissa27** [comment]
@martensonbj @Tman22

File: tests/unit/models/reminder-test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('reminder', 'Unit | Model | reminder', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
+
```
Comment (Tman22) [inline tests/unit/models/reminder-test.js]: Take this out if you're not actually going to test it.

**Comment by marissa27** [comment]
@Tman22 @martensonbj

**Comment by Tman22** [comment]
Nice work

**Pull request merged**